src/reconstruction/CoreModel.jl

```diff
@@ -397,18 +395,19 @@ end
 end
 
 @_remove_fn metabolite CoreModel Int plural inplace begin
-    remove_reactions!(
-        model,
-        [
-            ridx for ridx in 1:n_reactions(model) if
-            any(in.(findnz(model.S[:, ridx])[1], Ref(metabolite_idxs)))
-        ],
-    )
+    if keep_consistent
```

Comment:
this will need extra argument processing in remove_fn right?

Reply:
I just left this